apps/devhub/urls.py

```diff
@@ -156,8 +156,8 @@
     # Submission API
     url('^addon/submit/agreement/$', views.api_key_agreement,
         name='devhub.api_key_agreement'),
-
     url('^addon/api/key/$', views.api_key, name='devhub.api_key'),
+    url('^addon/api/creds/$', views.api_key_creds, name='devhub.api_creds'),
```

Comment:
I think calling this `addon/api/secret` would make more sense since it's not returning all credentials.